qa/workunits/rbd/import_export.sh

```diff
@@ -74,70 +74,72 @@ cmp ${TMPDIR}/img ${TMPDIR}/img3
 
 rm ${TMPDIR}/img ${TMPDIR}/img2 ${TMPDIR}/img3
 
-# try with --export-format for snapshots
-dd if=/bin/dd of=${TMPDIR}/img bs=1k count=10 seek=100
-rbd import $RBD_CREATE_ARGS ${TMPDIR}/img testimg
-rbd snap create testimg@snap
-rbd export --export-format 2 testimg ${TMPDIR}/img_v2
-rbd import --export-format 2 ${TMPDIR}/img_v2 testimg_import
-rbd info testimg_import
-rbd info testimg_import@snap
-
-# compare the contents between testimg and testimg_import
-rbd export testimg_import ${TMPDIR}/img_import
-compare_files_and_ondisk_sizes ${TMPDIR}/img ${TMPDIR}/img_import
-
-rbd export testimg@snap ${TMPDIR}/img_snap
-rbd export testimg_import@snap ${TMPDIR}/img_snap_import
-compare_files_and_ondisk_sizes ${TMPDIR}/img_snap ${TMPDIR}/img_snap_import
-
-rm ${TMPDIR}/img_v2
-rm ${TMPDIR}/img_import
-rm ${TMPDIR}/img_snap
-rm ${TMPDIR}/img_snap_import
-
-rbd snap rm testimg_import@snap
-rbd remove testimg_import
-rbd snap rm testimg@snap
-rbd rm testimg
-
-# order
-rbd import --order 20 ${TMPDIR}/img testimg
-rbd export --export-format 2 testimg ${TMPDIR}/img_v2
-rbd import --export-format 2 ${TMPDIR}/img_v2 testimg_import
-rbd info testimg_import|grep order|awk '{print $2}'|grep 20
-
-rm ${TMPDIR}/img_v2
-
-rbd remove testimg_import
-rbd remove testimg
-
-# features
-rbd import --image-feature layering ${TMPDIR}/img testimg
-FEATURES_BEFORE=`rbd info testimg|grep features`
-rbd export --export-format 2 testimg ${TMPDIR}/img_v2
-rbd import --export-format 2 ${TMPDIR}/img_v2 testimg_import
-FEATURES_AFTER=`rbd info testimg_import|grep features`
-if [ "$FEATURES_BEFORE" != "$FEATURES_AFTER" ]; then
-  false
-fi
+if rbd help export | grep -q export-format; then
```

Comment:
this change fixes the failure of http://pulpito.ceph.com/kchai-2017-06-22_00:11:35-upgrade-master---basic-mira/

@dillaman could you help review this commit?